signing/fix_codesign_issues.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+pushd Cura.app/Contents/MacOS
+
+dirs=`find . -type d -depth 1`
+
+for dir in $dirs; do
+    filename=`basename $dir`
+    mv "$dir" "../Resources/"
+    ln -s "../Resources/$filename" .
+done
+
+popd
```

Comment:
This is all rather arcane. Perhaps you could add a bit of documentation what this is for and what the problem was? Why are we moving these directories?